### PR TITLE
fix: upgrade to Elm 0.19.1

### PR DESCRIPTION
**Pull request by mbylstra**
This PR upgrades from 0.19.0 to 0.19.1. The main change here is that the Elm compiler is much stricter with how the elm module names like `module ElmStories.TagStories exposing (main)` match the directory structure. We can no longer use the existing `docs/Thing.stories.elm` structure, so each package has a new directory named `ElmStories` at the root of the package (it can't be put inside `docs` any more). Also this uses an upgraded version of `cultureamp/elm-storybook` that's compatible with Elm 0.19.1 (and requires a bit more boilerplate to use)

Here is the Chromatic build: https://www.chromatic.com/build?appId=5e12c297b9d9020020789ca3&number=234. I'm pretty sure the changes there are just the baseline snapshots are a little behind master.

### CODE REVIEW

File: draft-packages/illustration/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "@kaizen/hosted-assets": "^1.1.0"
   },
   "devDependencies": {
-    "@cultureamp/elm-storybook": "cultureamp/elm-storybook#0.1.0",
+    "elm-storybook": "cultureamp/elm-storybook#0.1.0",
```
Comment (jasononeil) [inline draft-packages/illustration/package.json:38]: Should this be 0.3.0?

It seems this was the only package still on 0.1.0 so may have been missed in a find/replace

Comment (jasononeil) [reply draft-packages/illustration/package.json]: It wasn't actually used in this package (no Elm version, I'm guessing it just came from the boilerplate). I'm going to bump it for consistency.

**Review by jasononeil — APPROVED** [review]
How I am reviewing:

- [x] Checked changes to package.json
- [x] Checked all other changes are to Elm storybook files
- [x] Visually checked Elm storybook
- [ ] Get Chromatic to check differences

I'm still learning more about how to do that last step properly, but I'm confident this is safe to merge.

**Review by jasononeil — DISMISSED** [review]
Reapprove after rebase

File: .tool-versions [hunk 2]
```diff
@@ -1,3 +1,3 @@
 nodejs 14.15.1
 yarn 1.17.2
-elm 0.19.0
+elm 0.19.0
```
Comment (mbylstra) [inline .tool-versions:3]:
```suggestion
elm 0.19.1
```

**Review by jasononeil — APPROVED** [review]

**Pull request merged**